osn.py

```
@@ -66,7 +66,8 @@ def test_block(BLOCKLIST):
             server = socketserver.UDPServer((HOST, PORT), WebBlock)
             print(f"Running")
             server.serve_forever()
-
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)